scss/_functions.scss

```diff
@@ -53,13 +56,28 @@
   $r: red($color);
   $g: green($color);
   $b: blue($color);
+  // get the relative lum for each color
+  $luminositecouleur: relativelum($r / 255, $g / 255, $b / 255);
```

Comment:
actually, to stay consistent with current WCAG wording (though from what I remember there's discussion about it being incorrect), this should be `$luminancecolor` and `$luminancewhite`